app/models/notification.rb

```diff
@@ -105,10 +115,16 @@ def send_welcome_notification_without_delay(receiver_id)
     end
 
     def send_moderation_notification(notifiable)
+      # TODO: make this work for articles in the future. only works for comments right now
+      return if UserBlock.exists?(blocker_id: notifiable.commentable.user_id, blocked_id: notifiable.user_id)
+
       Notifications::ModerationNotificationJob.perform_later(notifiable.id)
     end
 
     def send_moderation_notification_without_delay(notifiable)
+      # TODO: make this work for articles in the future. only works for comments right now
+      return if UserBlock.exists?(blocker_id: notifiable.commentable.user_id, blocked_id: notifiable.user_id)
```

Comment:
This code seems like something that could be extracted as a private method in the class...

```
UserBlock.exists?(blocker_id: notifiable.commentable.user_id, blocked_id: notifiable.user_id)
```

Since it's used over and over again.